traveller/modules/cfp/view.py

```diff
@@ -47,7 +47,7 @@ def add_talk(year):
         talk.author_list = AuthorList()
     talk.author_list.authors.append(current_user)
     talk.submitter_id = current_user.id
-    talk.year_submitted = str(dt.utcnow().year)
```

Comment:
Actually i was suggesting to change to

```
talk.year_submitted = year
```
since we are already passing in year as parameter

talk.year also sounds good, just change the other part